datamodel.itomig-webhook-integration.xml

```diff
@@ -281,63 +301,70 @@
           <static>false</static>
           <access>protected</access>
           <type>Custom</type>
-          <code><![CDATA[protected function preparePostData($aContextArgs, &$oLog)
-          {
-            $aSlackData = array(
-              'type' => 'mrkdwn',
-              'text' => empty($this->Get("text")) ? "" : $this->prepareTextForSlack(MetaModel::ApplyParams($this->Get("text"), $aContextArgs))
-            );
+          <code><![CDATA[	protected function preparePostData($aContextArgs, &$oLog)
+	{
+		$sText = empty($this->Get("text")) ? "" : $this->prepareTextForSlack(MetaModel::ApplyParams($this->Get("text"), $aContextArgs));
+		$aSlackData = array(
+			'text' => $sText,
+			'blocks' => array(
+				array(
+					'type' => 'section',
+					'text' => array(
+						'type' => 'mrkdwn',
+						'text' => $sText,
+					),
+				),
+			),
+		);
 
-            if (!empty($this->Get('channel'))){
-              $aSlackData['channel'] = $this->Get('channel');
-            }
-            if (!empty($this->Get('bot_alias'))){
-              $aSlackData['username'] = $this->Get('bot_alias');
-            }
-            
+		if (!empty($this->Get('channel'))) {
+			$aSlackData['channel'] = $this->Get('channel');
+		}
+		if (!empty($this->Get('bot_alias'))) {
+			$aSlackData['username'] = $this->Get('bot_alias');
+		}
 
-            $aSlackBlocks = array();
+		// Fields from list view
+		if ($aFields = $this->GetListViewAttributes($aContextArgs)) {
+			$aSlackData['blocks'][] = array(
+				'type' => 'section',
+				'fields' => $aFields,
+			);
+		}
 
-            // Add basic text also as section
-            if($aFields = $this->GetListViewAttributes($aContextArgs)){
-              $aSlackBlocks[] = array(
-                'type' => 'section',
-                'text' => array(
-                  'type' => isset($aSlackData['type']) ? $aSlackData['type'] : 'mrkdwn',
-                  'text' => isset($aSlackData['text']) ? $aSlackData['text'] : ''
-                )
-              );
-            }
+		/**
```

Comment:
Think we should make it configurable whether user info is displayed in the message or not